xml/operating_system.xml

```diff
@@ -315,6 +327,25 @@
     <param pos="1" name="os.version"/>
     <param pos="2" name="os.edition"/>
   </fingerprint>
+  <fingerprint pattern="^(?i:VMWare Photon(?:\/)?(?:\s?Linux)?\s?(?:v)?(\d+?(?:\.\d+?)*?)?)$">
+    <description>Photon Linux</description>
+    <example os.vendor="VMWare" os.product="Photon Linux">VMWare Photon Linux</example>
```

Comment:
It is fine to test this value, but there is no need to test `os.vendor` or `os.product` in this case because you hardcoded them.